src/GafferImage/ImageWriter.cpp

```diff
@@ -869,6 +935,12 @@ IECore::MurmurHash ImageWriter::hash( const Context *context ) const
 void ImageWriter::execute() const
 {
 	// Create an OIIO::ImageOutput
+	Context * c = const_cast<Context *>( Context::current() );
```

Comment:
The context is const for a very good reason, so we should make a copy before editing it. I have a PR up that I hope will be merged today, which will make that even easier, looking like this :

```
Context::EditableScope colorSpaceScope( Context::current() );
colorSpaceScope.set( "gaffer:colorspace", defaultColorSpace()( currentFileFormat() );
```